ultrasonic/src/main/java/org/moire/ultrasonic/util/Util.java

```diff
@@ -606,6 +620,18 @@ public void onClick(DialogInterface dialog, int i)
 		}).show();
 	}
 
+	public static void showDialog(Context context, int icon, int titleId, String message)
+	{
```

Comment:
Try to use applicationContext() instead of passing around a context

Reply:
I might need help on how to do that here

Reply:
I was actually wrong! Unlike toast, AlertDialog actually requires the Activity context!

But I cleaned up the methods a bit (see my commit, hope that was ok)